jwst/lib/set_telescope_pointing.py

```diff
@@ -441,7 +453,7 @@ def update_wcs_from_telem(
          model.meta.wcsinfo.pc1_2,
          model.meta.wcsinfo.pc2_1,
          model.meta.wcsinfo.pc2_2) = calc_rotation_matrix(
-             wcsinfo.pa * D2R, vparity=siaf.vparity
+             wcsinfo.pa * D2R, np.deg2rad(model.meta.wcsinfo.v3yangle), vparity=siaf.vparity
```

Comment:
Since `D2R` is likely almost exactly the same as `np.deg2rad`, could we use one or the other here?